cvelib/cli.py

```diff
@@ -123,14 +124,25 @@ def handle_cve_api_error(func: Callable) -> Callable:
     def wrapped(*args: Any, **kwargs: Any) -> Callable:
         try:
             return func(*args, **kwargs)
-        except CveApiError as exc:
-            error, _, details = str(exc).partition("; returned error: ")
-            click.secho("ERROR: ", bold=True, nl=False)
-            click.echo(error)
-            if details:
-                click.secho("DETAILS: ", bold=True, nl=False)
-                click.echo(details)
-            sys.exit(1)
+        except requests.exceptions.ConnectionError as exc:
+            error = str(exc)
+        except requests.exceptions.RequestException as exc:
+            if exc.response is not None:
+                try:
+                    error = exc.response.json()
+                except ValueError:
+                    error = exc.response.content
+                error = f"{exc}; returned error: {error}"
```

Comment:
I guess this could be skipped since you're just parsing it out below into the two individual parts again. You could set the `.json()` or `.content` values as details right from the start with a default of `details = None`.

Reply:
Yeah, see above - I didn't want to introduce a regression so didn't bother to simplify, but I could if you'd like.

Reply:
Yea, let's simplify! We'll be bumping the major version for the next release so this is our chance to introduce breaking changes :-)